### PR TITLE
docs: fix small style / indent bugs

### PR DESCRIPTION
**Pull request by luwes**

### CODE REVIEW

File: docs/src/components/Sandpack.jsx [hunk 1]
```diff
@@ -35,6 +35,10 @@ export default function ComponentSandpack({
       template="vanilla"
       theme={{
         ...githubLight,
+        syntax: {
+          ...githubLight.syntax,
+          definition: '#22863a'
```
Comment (luwes) [inline docs/src/components/Sandpack.jsx:40]: the tag color of github light is green, not purple

File: docs/src/pages/en/get-started.mdx [hunk 2]
```diff
@@ -29,23 +29,23 @@ This example demonstrates combining controls, captions, poster image, and a time
...
+    <media-play-button></media-play-button>
+    <media-seek-backward-button></media-seek-backward-button>
+    <media-seek-forward-button ></media-seek-forward-button>
+    <media-mute-button></media-mute-button>
+    <media-time-range></media-time-range>
+    <media-time-display></media-time-display>
+    <media-captions-button></media-captions-button>
+    <media-fullscreen-button></media-fullscreen-button>
+  </media-control-bar>
+</media-controller>`}
```
Comment (luwes) [inline docs/src/pages/en/get-started.mdx:48]: indent issue

File: docs/src/pages/en/get-started.mdx [hunk 3]
```diff
@@ -131,8 +131,9 @@ You can add controls to your player by placing them inside of a `<media-control-
 When using controls outside of a `<media-controller>` wrapper element; the control, or the surrounding control bar, needs to specify which media controller it should be associated with. To do this you can set the `mediacontroller` attribute.
 
 <SandpackContainer
+  height={244}
```
Comment (luwes) [inline docs/src/pages/en/get-started.mdx:134]: this Sandpack preview had an unneeded scrollbar

File: docs/src/styles/index.css [hunk 4]
```diff
@@ -320,6 +320,11 @@ table {
   border-collapse: collapse;
 }
 
+pre {
+  -webkit-font-smoothing: antialiased;
+  font-size: .8125em; /* 13px */
+}
```
Comment (luwes) [inline docs/src/styles/index.css:326]: this makes the inline code blocks more similar to the Sandpack code blocks

File: docs/src/styles/theme.css [hunk 5]
```diff
@@ -120,7 +120,7 @@ body {
   --theme-code-bg: hsla(var(--color-gray-5), 1);
   --theme-code-text: hsla(var(--color-base-white), 100%, 1);
   --theme-navbar-bg: rgb(56 56 56);
-  --theme-footer-bg: hsla(211, 29%, 14%, 1);
+  --theme-footer-bg: rgb(56 56 56);
```
Comment (luwes) [inline docs/src/styles/theme.css:123]: footer bg color was wrong in dark mode

**Review by cjpillsbury — APPROVED** [review]
don't see any concerns here

**Pull request merged**